requests/sessions.py

```diff
@@ -471,17 +471,17 @@ def request(self, method, url,
             params=None, data=None, headers=None, cookies=None, files=None,
             auth=None, timeout=None, allow_redirects=True, proxies=None,
             hooks=None, stream=None, verify=None, cert=None, json=None):
```

Comment:
The documentation has `json` after `data` and at least in `requests.api.request` there are no positional arguments except `url`.

As it is a breaking change to move `json` to the front of the list, I didn't change it. However, I think it might be something to think about:

```python
    def request(self, method, url, *,
            params=None, data=None, json=None, headers=None, cookies=None,
            files=None, auth=None, timeout=None, allow_redirects=True,
            proxies=None, hooks=None, stream=None, verify=None, cert=None):
```